### ISSUE TITLE
Support for python 3.13

### ISSUE THREAD

**Issue by Housebi** [CoolProp/CoolProp#2430]
After installing CoolProp with pip on python 3.13 it failed to import correctly.

While importing it raises a TypeError (expected bytes, str found). I will have to check on a different computer again if I can replicate the error as it is now solved by installing CoolProp with the source file and running python setup.py as described [here](https://stackoverflow.com/questions/74093424/how-to-install-coolprop-in-python3-9windows-and-pydroid3android).

From what I could find it could be that python 3.13 is not supported yet. This because the error looks similar to [#2310](https://github.com/CoolProp/CoolProp/issues/2310).

**CoolProp Version:** 6.6.0
**Operating System and Version:** Windows 10
**Access Method:** Python

### Additional Information

The error message after installing CoolProp with pip and running a random script that includes CoolProp.

Python 3.13.0 (tags/v3.13.0:60403a5, Oct  7 2024, 09:38:07) [MSC v.1941 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

= RESTART: C:\Users\Niels Holswilder\source\repos\CoolpropTest\CoolpropTest.py =
Traceback (most recent call last):
  File "C:\Users\Niels Holswilder\source\repos\CoolpropTest\CoolpropTest.py", line 1, in <module>
    import CoolProp.CoolProp as CP
  File "C:\Program Files\Python313\Lib\site-packages\CoolProp\__init__.py", line 22, in <module>
    __fluids__ = CoolProp.get_global_param_string('fluids_list').split(',')
  File "CoolProp.pyx", line 309, in CoolProp.CoolProp.get_global_param_string
  File "<stringsource>", line 15, in string.from_py.__pyx_convert_string_from_py_std__in_string
TypeError: expected bytes, str found

**Comment by ibell** [comment]
Yes python 3.13 is not yet supported. Are you volunteering to take that on?

**Comment by Housebi** [comment]
So I tried creating a wheel file for python 3.13, which at first glance seems to have worked. (After several tries). But I did need to edit some parts in the setup.py file as distutils seemed to cause an error. Perhaps it was because I also installed python 3.12 and my system set it to default. But there were no packages installed on this python 3.12 version. Or it is because distutils is no longer supported and should be replaced by [setuptools](https://peps.python.org/pep-0632/). But this does not explain why it did work under python 3.12, which it no longer should.

In any case, in the setup file I replaced instances of distutils. Then I created a wheel, by running python setup.py bdist_wheel.

I only tried it with one piece of code to test whether it works, which it seems to do.
